docs/RFCS/settings_table.md

```diff
@@ -0,0 +1,103 @@
+- Feature Name: Settings Table
+- Status: draft
+- Start Date: 2017-03-17
+- Authors: David Taylor
+- RFC PR:
+- Cockroach Issue:
+
+# Summary
+
+A system table of named settings with a caching accessor on each node to provide
+runtime-alterable execution parameters.
```

Comment:
I think we could very well call this the "cluster config". I think the scary part is 1. some parameters might need a node restart, 2. many customer would like this rolled out using a schedule rather than "immediately"